step.yml

```diff
@@ -1,26 +1,57 @@
 ---
-title: Artifact pull
-summary: Step to pull artifacts of a pipeline
+title: Pull Pipeline intermediate files
+summary: Pull intermediate files generated by the Pipeline Builds.
 description: |-
-  The step downloads build artifacts of a pipeline to a local folder.
+  The step downloads intermediate files (build artifacts generated by the pipeline workflows,
+  that are intended to be shared with subsequent workflows of the pipeline) to a local folder.
 
-  By default, all artifacts generated by any workflow of the pipeline are downloaded. This can be limited
-  by setting the `artifact_sources` input variable.
+  These artifacts can be uploaded by the [Deploy to Bitrise.io step](https://www.bitrise.io/integrations/steps/deploy-to-bitrise-io),
+  via specifying the *Files to share between pipeline stages* input.
+
+  Pipeline intermediate files have an environment variable key assigned,
+  after downloading the given artifact, an environment variable (with the assigned key) will point to the file's local path.
+
+  Directories specified in *Files to share between pipeline stages* input will be archived and uploaded as a single file.
+  After downloading, they are extracted, and the specified environment variable will point to the directory's local path.
+
+  By default, all artifacts generated by any workflow of the pipeline are downloaded.
+  This can be limited by setting the *Artifact source* input.
```

Comment:
Replace with:
### Configuring the Step
  To configure the Step:

1. Specify which artifacts you want to download in the **Artifact source** input. By default, all artifacts generated by the Workflows in the Pipeline will be downloaded. 
NOTE: You can list multiple artifacts by separating them using a comma. For example: `{stage1}.{workflow1},{stage2}.{workflow2}`
2. (Optional) Set the **Enable verbose logging** input to `true` if you want to log additional information for debugging purposes.

Reply:
> Specify which artifacts you want to download

What you specify here are workflows, which workflows' artifacts will be downloaded.